CIP-xxxxx/README.md

```diff
@@ -0,0 +1,208 @@
+
+Finally, the `signData()` endpoint will have to be disabled when using this extension since they are not compatible with Plutus-based addresses. Plutus contracts cannot sign data, only validate transactions.
+
+## Specification
+
+### Versioning
+
+To facilitate future updates, the API will be versioned. Newer version of the API have to be compatible with the previous versions. 
+
+The current version of the API is 2.0, the major version number is derived from the version of Plutus that the API implements, and the minor version number is used for incremental updates within the same version of Plutus.
+
+### Data Types
+
+#### KeyHash
+
+A hex-encoded string of the corresponding bytes. This represents the hash of the public key used to sign transactions.
+
+```ts
+type KeyHash = String
+```
+
+#### ScriptRequirements
+
+Script requirements encapsulate all the possible requirements for a transaction to be valid. It includes the following fields:
+
+- collateral: The list of inputs that will be used as collateral for the transaction.
+- inputs: The list of inputs that will be used as inputs for the transaction.
+- reference_inputs: The list of inputs that will be used as reference inputs for the transaction.
+- outputs: The list of outputs that will be used for the transaction.
+- mint: The amount of tokens that will be minted/burned in the transaction.
+- certificates: The list of certificates that will be included in the transaction.
+- withdrawals: The list of withdrawals that will be included in the transaction.
+- validity_range: The validity range that the transaction must be valid in.
+- signatories: The list of signatories that must sign the transaction.
+- redeemers: The list of redeemers that will be used in the transaction.
+- datums: The list of datums that will be used in the transaction.
+
+
+```ts
+type ScriptRequirement = {
+  collateral?: List<cbor<transaction_unspent_output>>,
+  inputs?: List<cbor<transaction_unspent_output>>,
+  reference_inputs?: List<cbor<transaction_unspent_output>>,
+  outputs?: List<transaction_output>,
+  mint?: Value,
+  certificates?: List<Certificate>,
+  withdrawals?: Dict<StakeCredential, Int>,
+  validity_range?: ValidityRange,
+  signatories?: List<KeyHash>,
+  redeemers?: Dict<ScriptPurpose, Redeemer>,
+  datums?: Dict<Hash<Blake2b_256, Data>, Data>
+}
+```
+
+#### ValidityRange
+
+```ts
+type ValidityRange = {
+  valid_before?: PosixTime,
+  valid_after?: PosixTime,
+}
+```
+
+#### Value
+```ts
+type Value = {
+    amount: Coin,
+    asset: String
+}
+```
+
+### Aditional Error Types
+
+#### CompletedTxError
+
+```ts
+CompletedTxErrorCode = {
+	NotFound: 1,
+	NotReady: 2
+}
+```
+
+* NotFound - The transaction with the given id was not found.
+* NotReady - The transaction with the given id is not ready yet. 
+
+## Motivation: why is this CIP necessary?
```

Comment:
Duplicate section

Reply:
Not really a duplicate section but fixed in https://github.com/cardano-foundation/CIPs/pull/798/commits/4805b23d537f3a8c3e35fdb7731ab27e53a5138a.